test_runner/src/main/kotlin/ftl/shard/Shard.kt

```diff
@@ -136,7 +136,8 @@ object Shard {
         testcases.sortByDescending { it.time }
 
         testcases.forEach { testMethod ->
-            val shard = shards.first()
+            // num_shards must be > 1, and <= 50
```

Comment:
just for clarification. num shards must be less than 50 or number of tests per shard must be less than 50?

Reply:
> The number of shards should be less than the total number of test cases. The number of shards specified must be >= 1 and <= 50.
>
> https://cloud.google.com/sdk/gcloud/reference/alpha/firebase/test/android/run

I think (hopefully) there's no limit on the number of tests per shard.

Reply:
I think this is >= 1, and <= 50